src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -0,0 +1,416 @@
+import React from "react";
+import * as shortid from "shortid";
+import { IAssetMetadata, IRegion, RegionType,
+        AssetState, EditorMode, IProject } from "../../../../models/applicationState";
+import { CanvasTools } from "vott-ct";
+import { Editor } from "vott-ct/lib/js/CanvasTools/CanvasTools.Editor";
```

Comment:
Let's add an item to the backlog for canvas tools so we don't need to put the paths down to all these sub areas.